index.html

```diff
@@ -8,7 +8,7 @@
 
     <h1>p5 sketches</h1>
 
-    <a href="template">template</a>
```

Comment:
u want to put this line back in, && add a `<br>` between ur link && this one, it should look like:
```html
<a href="template">template</a>
<br>
<a href="parker_davis">parker_davis</a>
```